x-pack/plugins/security_solution/common/endpoint/schema/trusted_apps.ts

```diff
@@ -29,8 +29,13 @@ export const GetTrustedAppsRequestSchema = {
   }),
 };
 
-const ConditionEntryTypeSchema = schema.literal('match');
-const ConditionEntryOperatorSchema = schema.literal('included');
+const ConditionEntryTypeSchema = schema.conditional(
+  schema.siblingRef('field'),
+  ConditionEntryField.PATH,
+  schema.oneOf([schema.literal('match'), schema.literal('wildcard')]),
+  schema.literal('match')
+);
+const ConditionEntryOperatorSchema = schema.literal('included' as ConditionEntry['operator']);
```

Comment:
I don't think you need the cast here either?

Reply:
done 0bf81993a3645cb4a493fa82e717bec675ba0ecc